src/components/UI/Ol.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
```

Comment:
Only decimal numbers are supported for now but it should be possible to make the component dynamic.